### ISSUE TITLE
[Adaptive Cards - Show Card Wizard] - While navigating on the 'Name' edit field, screen reader is not conveying the required information along with the label.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9016]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro steps:**
1. Open the above URL in the edge dev browser.
2. Navigate to the 'Sample' option and press enter key to activate it.
3. Press tab key and navigate to the 'Show Card Wizard' link and press enter key to activate it.
4. Press tab key and navigate the card and press tab key and navigate to 'Name' button.
5. Press enter key to expand it.
6. Now press tab key and navigate to the 'First Name' edit field and observe the issue.

**Actual Result:**
While navigating on the 'First Name' edit field screen reader is not announcing the required information. It is just announcing as **'First Name edit has a pop up.'**
Also, it is announcing irrelevant information as has pop up for the edit field.

**Expected Result:**
Screen Reader should announce the required information while navigating on the 'First Name' edit field also it should not announce the has a pop information for the edit field. It should announce something like **'First Name edit required.'**

**Note:**
Same issue is repro throughout the application for the required edit field.
Same issue is repro with NVDA.

**User Impact:**
Users rely on the screen reader to communicate whether a field is required or optional. If this information is not conveyed, users may leave required fields blank or spend time filling out optional fields unnecessarily.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

“ Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section”

### Screenshots

![While navigating on the First Name edit field, Screen reader is not conveying required information](https://github.com/user-attachments/assets/cf26ab0c-8855-483e-b078-eba41e829e2d)

https://github.com/user-attachments/assets/f9af82a3-2e6b-4c06-8a73-9cd1eb1def32

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Note: Input.Text is missing the aria-required option

**Comment by anna-dingler** [comment]
@vagpt This issue was fixed with PR #8938. I am closing this issue and will try to release a new version of our site this month. For now, I am marking this bug as not ready to test.

**Comment by anna-dingler** [comment]
We have released another version of the website, and this fix is available. Thanks!

**Comment by vagpt** [comment]
This issue is now working fine, hence closing this issue.

![Image](https://github.com/user-attachments/assets/f45f6232-98e1-43c0-b040-9eff239564ee)

![Image](https://github.com/user-attachments/assets/d876d5ec-fa62-447b-a6fc-e2f65368538f)
